packages/teams-js/src/internal/handlers.ts

```diff
@@ -155,7 +156,7 @@ export function registerHandlerHelper(
  */
 export function registerOnThemeChangeHandler(handler: (theme: string) => void): void {
   HandlersPrivate.themeChangeHandler = handler;
-  handler && sendMessageToParent('registerHandler', ['themeChange']);
+  !isNullOrUndefined(handler) && sendMessageToParent('registerHandler', ['themeChange']);
```

Comment:
>!isNullOrUndefined(handler) [](http://example.com/codeflow?start=2&length=27)

Without this, the compiler will warn you because it thinks you are trying to call `handler()` but forgot the parenthesis. In this case, the code is using the `handler` structure to validate that it has a real value.